cypress/integration/common/report.js

```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('title').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.base-card .title').should('contain', annoyingUserWhoMutedModeratorTitle)
```

Comment:
not related to this `PR` directly, but not even sure how this was passing!? the `title` doesn't have a `.` indicating it's a class :woman_shrugging: